### ISSUE TITLE
Allow local copy from darktable-cli

### ISSUE THREAD

**Issue by webysther** [darktable-org/darktable#16253]
**Is your feature request related to a problem? Please describe.**
N/A

**Describe the solution you'd like**
There a way to make a local copy of entire collection from darktable-cli? I want to automate the ingestion, thumb and local copy.

**Comment by MStraeten** [comment]
why using cli for that? the images are on your local drives - you don't need darktable to copy

**Comment by webysther** [comment]
> @MStraeten : We are speaking of `local copies` a way to have a copy the RAW that to work with but still referencing the original one.
> 
> https://docs.darktable.org/usermanual/development/fr/overview/sidecar-files/local-copies/

Yes, My main ideia is when in the field making pictures to wikipedia is able to upload to the NAS and run the job to application server local to copy the files as a Hot cache, I will use the darktable version of linuxserver to access the application over browser vnc (kasmvnc). With local copies the thumb creation become really fast in comparison with creation over NFS.

**Comment by webysther** [comment]
As a volunteer now I don't have much time to invest organizing and I look ways to automate the process, I have more images than I can adjust to upload to commons, with digikam upload to wikimedia is really nice but for make raw edit darktable is king, the biggest problem I have is working and keep the process out of my notebook to scale the hardware if necessary and make easy to move between countries without lost the ability to process those images when I'm moving or working.

**Comment by webysther** [comment]
> > With local copies the thumb creation become really fast in comparison with creation over NFS.
> 
> If you activate the caching the thumb is done once, so not a so big deal.
> 
> Also, having `darktable-cli` to create local copies is a separate issue than having the thumb caching populated as the later is already supported by `darktable-generate-cache`. All in all I'm not sure about the usefulness of this request.

The local copy don't have any reference in database/xmp right? (https://github.com/darktable-org/darktable/blob/master/src/common/image.c#L2901)

If this is true I can process using a external simple script, correct?

**Comment by webysther** [comment]
Look here please:

https://github.com/darktable-org/darktable/issues/16252#issuecomment-1932537646

**Comment by webysther** [comment]
Moved to #16252
